src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import App from './App.js';
```

Comment:
There is no need of adding the file extension while importing that file. It was correct before. :+1: